README.md

```diff
@@ -1,17 +1,7 @@
 
 # Knative Serving Operator
 
-The following will install [Knative Serving](https://github.com/knative/serving)
-and configure it appropriately for your cluster in the `knative-serving`
-namespace:
-
-```
-kubectl apply -f deploy/crds/serving_v1alpha1_knativeserving_crd.yaml
-kubectl apply -f deploy/
-```
-
-To be clear, the operator will be deployed in the `default` namespace, and then
-it will install Knative Serving in the `knative-serving` namespace.
+Knative Serving Operator is a project aiming to deploy and manage Knative Serving in an automated way.
```

Comment:
Does anyone have a better description of serving operator? Welcome the comments.